src/presenters/title-presenter.php

```diff
@@ -37,17 +37,21 @@ public function __construct( String_Helper $string ) {
 	 * Returns the document title.
 	 *
 	 * @param Indexable_Presentation $presentation The presentation of an indexable.
+	 * @param bool                   $output_html  Optional. Whether or not to output HTML. Defaults to true.
 	 *
 	 * @return string The document title tag.
 	 */
-	public function present( Indexable_Presentation $presentation ) {
+	public function present( Indexable_Presentation $presentation, $output_html = true ) {
```

Comment:
`$output_title_tags` may be a more descriptive argument name.
The argument documentation could then be:
```
* @param bool                   $output_title_tags Optional. Whether or not to output the title with the title tags (`<title>`) included. Defaults to true.
```